lib/fhir_models/bootstrap/generator.rb

```diff
@@ -199,7 +199,7 @@ def generate_class(hierarchy, structure_def, top_level = false)
                 binding_uri = binding_uri[0..-7] if binding_uri&.end_with?('|4.0.0')
                 codes = @defn.get_codes(binding_uri)
                 field.valid_codes = codes unless codes.nil?
-                if field.valid_codes.empty? && field.binding['uri'] && !binding_uri.end_with?('bcp47') && !binding_uri.end_with?('bcp13.txt')
+                if field.valid_codes.empty? && binding_uri && ['bcp47', 'bcp13.txt', 'mimetypes'].none? { |x| binding_uri.end_with?(x) }
```

Comment:
Adjusted based on your suggestion, but had to leave in the extra `&& binding_uri` in the middle because I don't want nulls in there and the last clause is actually a **not** (`!binding_uri.end_with?`)